Registry/Registry.EM_COMMON

```diff
@@ -2453,6 +2453,7 @@ rconfig   integer ISFFLX                  namelist,physics 	1             1
 rconfig   integer IFSNOW                  namelist,physics	1             1       irh    "IFSNOW"                        ""      ""
 rconfig   integer ICLOUD                  namelist,physics	1             1       irh    "ICLOUD"                        ""      ""
 rconfig   integer cldovrlp                namelist,physics      1             2       irh    "cldovrlp"                      "1=random, 2=maximum-random, 3=maximum, 4=exponential, 5=exponential-random"      ""
+rconfig   integer idcor                   namelist,physics      1             0       irh    "idcor"                         ""      ""
```

Comment:
In the unused part of the nml, instead of the first empty quote, add "0=constant decorr len=2500 m, 1=latitude varying decorr len". I tend to look in the Registry before looking in the README.namelist for a quick reference.